tests/modules/programs/k9s/empty-settings.nix

```
@@ -0,0 +1,13 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.k9s.enable = true;
+
```

To avoid an unnecessary download of the k9s package:

```suggestion

  test.stubs.k9s = { };

```